exporter/metric/option.go

```diff
@@ -43,10 +44,20 @@ type options struct {
 	// metricDescriptorTypeFormatter is the custom formtter for the MetricDescriptor.Type.
 	// By default, the format string is "workload.googleapis.com/[metric name]".
 	metricDescriptorTypeFormatter func(metricdata.Metrics) string
+
+	// monitoredResourceCreator is the custom function that creates monitored resource
+	// for Google Cloud Monitoring
+	monitoredResourceCreator func(*metricdata.ResourceMetrics) (*monitoredrespb.MonitoredResource, error)
+
 	// resourceAttributeFilter determinies which resource attributes to
 	// add to metrics as metric labels. By default, it adds service.name,
 	// service.namespace, and service.instance.id.
 	resourceAttributeFilter attribute.Filter
+
+	// metricAttributesFilter determinies which metric attributes to
+	// add to metrics as metric labels
+	metricAttributesFilter attribute.Filter
```

Comment:
You shouldn't need this config in the exporter.  Use https://pkg.go.dev/go.opentelemetry.io/otel/sdk/metric#WithView to pass views to the metrics SDK, which allow filtering attributes on metrics.

Reply:
with views, the metric attributes do not reach the exporter at all. The attributes need to reach the exporter. Then, the exporter can add them to resource